app/src/main/res/layout/fragment_benefits.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@sandarumk 
This file is the layout file for **Benefits of seeking staff support** screen. 
There is another layout file for **PCs commitment** (fragment_commitment.xml) which has a similar structure. Should I use one XML file common to both ( or anymore screens which are strcuturally the same, to that extent ) or should I keep them as seperate layouts since they are different subsections altogether?